pkg/planner/core/casetest/tpch/tpch_test.go

```diff
@@ -0,0 +1,109 @@
+		"            └─ExchangeReceiver_141 39998176.29 mpp[tiflash]  ",
+		"              └─ExchangeSender_140 39998176.29 mpp[tiflash]  ExchangeType: HashPartition, Compression: FAST, Hash Cols: [name: test.lineitem.l_orderkey, collate: binary], [name: test.orders.o_orderdate, collate: binary], [name: test.orders.o_shippriority, collate: binary]",
+		"                └─HashAgg_136 39998176.29 mpp[tiflash]  group by:Column#48, Column#49, Column#50, funcs:sum(Column#47)->Column#43",
+		"                  └─Projection_145 92822759.11 mpp[tiflash]  mul(test.lineitem.l_extendedprice, minus(1, test.lineitem.l_discount))->Column#47, test.lineitem.l_orderkey->Column#48, test.orders.o_orderdate->Column#49, test.orders.o_shippriority->Column#50",
+		"                    └─Projection_125 92822759.11 mpp[tiflash]  test.orders.o_orderdate, test.orders.o_shippriority, test.lineitem.l_orderkey, test.lineitem.l_extendedprice, test.lineitem.l_discount",
+		"                      └─HashJoin_124 92822759.11 mpp[tiflash]  inner join, equal:[eq(test.orders.o_orderkey, test.lineitem.l_orderkey)]",
+		"                        ├─ExchangeReceiver_53(Build) 22867441.30 mpp[tiflash]  ",
+		"                        │ └─ExchangeSender_52 22867441.30 mpp[tiflash]  ExchangeType: HashPartition, Compression: FAST, Hash Cols: [name: test.orders.o_orderkey, collate: binary]",
+		"                        │   └─Projection_51 22867441.30 mpp[tiflash]  test.orders.o_orderkey, test.orders.o_orderdate, test.orders.o_shippriority",
+		"                        │     └─HashJoin_44 22867441.30 mpp[tiflash]  inner join, equal:[eq(test.customer.c_custkey, test.orders.o_custkey)]",
+		"                        │       ├─ExchangeReceiver_48(Build) 1501762.80 mpp[tiflash]  ",
+		"                        │       │ └─ExchangeSender_47 1501762.80 mpp[tiflash]  ExchangeType: Broadcast, Compression: FAST",
+		"                        │       │   └─TableFullScan_45 1501762.80 mpp[tiflash] table:customer pushed down filter:eq(test.customer.c_mktsegment, \"AUTOMOBILE\"), keep order:false",
+		"                        │       └─TableFullScan_49(Probe) 36377904.50 mpp[tiflash] table:orders pushed down filter:lt(test.orders.o_orderdate, 1995-03-13 00:00:00.000000), keep order:false",
+		"                        └─ExchangeReceiver_57(Probe) 162359270.28 mpp[tiflash]  ",
+		"                          └─ExchangeSender_56 162359270.28 mpp[tiflash]  ExchangeType: HashPartition, Compression: FAST, Hash Cols: [name: test.lineitem.l_orderkey, collate: binary]",
+		"                            └─TableFullScan_54 162359270.28 mpp[tiflash] table:lineitem pushed down filter:gt(test.lineitem.l_shipdate, 1995-03-13 00:00:00.000000), keep order:false"))
+}
```

Comment:
using MustQuery is hard to maintain when the diff is large

Reply:
This test case will be moved into this PR which optimize the MPP exchangement.

Reply:
It has been deleted now.